library/lib/node-utils.js

```diff
@@ -17,16 +24,74 @@ function evalWithSourceMap(source, createMap) {
 }
 
 function withSourceMappedError(createMap, fn, options) {
-  return withRawErrorStack(() => {
-    try {
-      return fn()
-    } catch (e) {
-      const messageWithStack = e + '\n' + toMappedStack(createMap, e.stack, options)
-      const error = new Error(messageWithStack)
-      error.stack = messageWithStack
-      throw error
-    }
-  })
+  try { return fn() } catch (e) {
```

Comment:
It seems we need a `addToSourceMapInfo` here, the `prepareStackTrace` depends on it. We however might not need it in a non-async environment.